packages/sanity/src/core/form/inputs/files/FileInput/FileActionsMenu.tsx

```diff
@@ -132,6 +132,7 @@ export function FileActionsMenu(props: Props) {
               mode="bleed"
               onClick={handleClick}
               ref={setOptionsButtonRef}
+              tooltipProps={{content: 'Open file options menu'}}
```

Comment:
```suggestion
              tooltipProps={{content: 'Show more'}}
```

If only to be consistent with other context menu buttons